_posts/2017-11-03-2017-crystal-survey-2017-results.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Crystal Survey 2017 Results"
+author: sdogruyol
+summary: ""
+thumbnail: ""
+---
+
+This year, the Crystal team launched the first official [State of Crystal survey](https://crystal-lang.org/2017/07/31/launching-the-2017-state-of-crystal-survey.html). We're amazed by the fantastic feedback from the community.
+
+We've received ***757*** responses, thanks a lot to everyone who took the time to respond.  
+
+We've tried our best to read through all of your comments and suggestions. Without further ado, let's take a look at the results.
+
+
+![%19.9 Google or other search engines, %19.4 HackerNews, %15.6 Reddit %10.9 Twitter](https://cdn.pbrd.co/images/GP30DmW.png
+  "%19.9 Google or other search engines, %19.4 HackerNews, %15.6 Reddit %10.9 Twitter")
+(hover for more info)
+
+We wanted to learn how did you discover Crystal, it's interesting to see that a lot our respondents find Crystal from search engines. HackerNews, Reddit and Twitter shows the power of social media.
+
+# Using Crystal
+
+![%47,2 Yes, I' using Crystal for hobby projects, %32.1 No, but I' planning to use Crystal, %10.7 No, I'm not using Crystal, %9.9 Yes, I'm Using Crystal in production](https://cdn.pbrd.co/images/GP33qmx.png
+"%47,2 Yes, I' using Crystal for hobby projects, %32.1 No, but I' planning to use Crystal, %10.7 No, I'm not using Crystal, %9.9 Yes, I'm Using Crystal in production")
+(hover for more info)
+
+It's amazing to see a lot people already using Crystal. Nearly half of our respondents use Crystal in for hobby projects. Another interesting point is that nearly 1 out of 10 are running in production.

```

Comment:
This is not a "verbatim" suggestion, I'm just tossing some lines of thought here. Let's continue asking ourselves what surprised us and what was kind of expected in each section. Some facts to highlight here: 

- There's a considerable number of people (42.9%) who's not using Crystal at all but still took the time to complete the survey, why?
- We sort of confirmed our intuition that at this point Crystal is mostly a hobby language.
- We're *psyched* that almost 10% of the respondents are already using Crystal in production. We want them to raise their hands, tell us how it's going for them and invite them to post in our blog.
- There's a deeper story that we see in these numbers, which is consistent with what we hear in informal discussions about Crystal: the community is eager to start using the language in production, but is waiting for more concrete signals from the project. A considerable part of the community doesn't even mind about that and trusts the ecosystem enough to take the leap.